### PR TITLE
Populate driver col in conv_config

### PR DESCRIPTION
**Pull request by alexandraBara**
Populating driver col in conv_config + restored pylint for import_configs

### CODE REVIEW

File: tuna/miopen/driver/convolution.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def compose_tensors(self, keep_id: bool = False) -> dict:
 
   def get_conv_dict(self) -> dict:
     """Populate c_dict with conv table elems"""
-    c_dict: Dict[str, int] = {}
+    c_dict: Dict[str, Any] = {}
```
Comment (technicalgrp89) [inline tuna/miopen/driver/convolution.py:171]: What is the reason to define it as Any from integer? reason?

File: tuna/miopen/driver/convolution.py [hunk 2]
```diff
@@ -177,6 +177,7 @@ def get_conv_dict(self) -> dict:
 
     c_dict['input_tensor'] = super().get_input_t_id()
     c_dict['weight_tensor'] = super().get_weight_t_id()
+    c_dict['driver'] = str(self)
```
Comment (technicalgrp89) [inline tuna/miopen/driver/convolution.py:180]: whats the actual value we get here as self object?

File: tuna/miopen/driver/convolution.py (hunk 1 above)
Comment (alexandraBara) [reply tuna/miopen/driver/convolution.py:171]: its got  a string as a value now as well, they are not all integers anymore

Comment (technicalgrp89) [reply tuna/miopen/driver/convolution.py:171]: ok

**Review by technicalgrp89 — APPROVED** [review]

File: tuna/miopen/subcmd/import_configs.py [hunk 3]
```diff
@@ -180,7 +180,8 @@ def import_cfgs(args: argparse.Namespace, dbt: MIOpenDBTables,
         parse_line(args, line, counts, dbt, logger)
       except ValueError as err:
         logger.warning(err)
-        continue
+        if 'Skipping line' in str(err):
+          continue
```
Comment (cderb) [inline tuna/miopen/subcmd/import_configs.py]: Continue looks like  a null op here.

File: tuna/miopen/utils/metadata.py [hunk 4]
```diff
@@ -220,6 +220,7 @@
     'FP16': 'FP16',
     'convfp16': 'FP16',
     'BF16': 'BF16',
+    'BFP16': 'BF16',
```
Comment (cderb) [inline tuna/miopen/utils/metadata.py]: Is it anticipated that Tuna encounters BFP16 as input? I thought it was intended to eliminate this.

**Review by cderb — APPROVED** [review]

**Pull request merged**